compiler/passes/convert-uast.cpp

```diff
@@ -3972,6 +3972,7 @@ Type* Converter::convertType(const types::QualifiedType qt) {
     case typetags::IntType:   return convertIntType(qt);
     case typetags::RealType:   return convertRealType(qt);
     case typetags::UintType:   return convertUintType(qt);
+    case typetags::CPtrType:   return dtCPtr;
```

Comment:
That's odd, I would have expected this would need to handle instantiations e.g. `c_ptr(int)`. Where would the `int` part be converted in such a case?

Reply:
Yes, you're right; I updated this mirror the (unimplemented) case of `convertClassType`.